Azure/azure-cli
# {Redis} `az redis update`: Support `--no-wait` to avoid breaking change and default to True

**Related command**
az redis update

**Description**
Resolves issue discussed here https://github.com/Azure/azure-cli/pull/24979#discussion_r1450185925

**Testing Guide**
Examples:
`az redis update -n {name} -g {rg} --set redisConfiguration.aadEnabled=true`- `az redis update` should return without waiting for result of long running operation
`az redis update -n {name} -g {rg} --set redisConfiguration.aadEnabled=false --no-wait False`- option to wait for result of long running operation, for functional test usage

---

This checklist is used to make sure that common guidelines for a pull request are followed.
